content/en/examples/controllers/nginx-deployment.yaml

```diff
@@ -19,3 +19,8 @@ spec:
         image: nginx:1.14.2
         ports:
         - containerPort: 80
+  strategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxSurge: 1
+    maxUnavailable: 1
```

Comment:
The indentation here doesn't look right.

Reply:
Thanks. Have updated the indentation.